### PR TITLE
feat(business): adapt usermenu styling

### PR DESCRIPTION
**Pull request by vlanz**
feat(usermenu): style business usermenu and add examples

Limit usermenu styling to not include 4k and 5k resolutions in business build and add showcase.

### CODE REVIEW

File: projects/angular-showcase/src/app/business/business-examples/usermenu-showcase/usermenu-showcase.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-12">
+      <p>
+        Please, to click on "Login" to login and subsequently to click on arrow down to expand the
```
[the remark this answers is not in the document]
Comment (vlanz) [reply projects/angular-showcase/src/app/business/business-examples/usermenu-showcase/usermenu-showcase.component.html]: Fixed here as well as in the public showcase.

**Comment by vlanz** [comment]
> I can't comment on code not affected by the PR, so here a manual comment.
> 
> _usermenu.scss:
> 
> * Border is not compliant with specification. The border in the specification has a width of 1px. Also change the transparent border width to 1px.
> * The color of the border for the business usermenu is `$sbbColorGranite`, not `$sbbColorGrey`.
> * There are a few 4k/5k blocks that should also be wrapped by publicOnly.
> 
> In order to apply the different values based on $sbbBusiness, you can either use the if block approach or the if assignment approach:
> 
> ```css-scss
> // If block
> $border: red solid 2px;
> @if $sbbBusiness {
>   $border: red solid 1px;
> }
> 
> // If assginment
> $border: if($sbbBusiness, red solid 1px, red solid 2px);
> ```

Thank you for the insight. I think the latest commits should address all your commented concerns; tell me if they don't! Can I ask where the border color rule was described? I couldn't find it in the UX specs.

**Comment by kyubisation** [comment]
@vlanz Thank you for your work so far 😃 
The design specification is mostly defined in the "Vermassungen" section links. In this case: https://sbb.invisionapp.com/d/main#/console/17140415/355318618/inspect
You can see here that the border is 1px, the top left and right corners have a radius of 2px and the border bottom is 1px solid $sbbColorCloud (#E5E5E5)

**Comment by vlanz** [comment]
@kyubisation I hope I got all the details this time... otherwise obviously I'll check again :)

**Comment by kyubisation** [comment]
LGTM 😃 
Thank you very much for your contribution!

**Pull request merged**